VeroliBGV/BGV009.xml

```diff
@@ -0,0 +1,1084 @@
+                     <msItem xml:id="p2_i1">
+                        <locus from="76ra" to="77ra"/>
+                        <title type="incomplete" ref="LIT3073RepCh353"/>
+                        <textLang xml:lang="gez"/>
+                        <note>The text is written in <ref target="#h2"/> and is the continuation of <ref target="p1_i3"/>, which begins on <locus target="#75ra"/>.
+                           The concluding supplication is seemingly incomplete.</note>
+                        <incipit xml:lang="gez">
+                           <locus target="#76ra"/>
+                           ልቀላ። አነ፡ ወአነ፡ <surplus>የ፡</surplus> <add place="interlinear">እቀድሞ፡</add> እንዘ፡ ይትበኃላ፡ <hi rend="rubric">ኤዎስጣቴዎስ<supplied reason="omitted" resp="MV">፡</supplied></hi>
+                           ኪያከ፡ ከመ፡ ይትቀበላ፡ ተፈጸመ፡ ብከ፡
+                        </incipit>
+                        <explicit xml:lang="gez">
+                           <locus target="#77ra"/>
+                           እንዘ፡ ትትኬነስ፡ ደሴተ፡ ቆጵሮስ፡ ለብፁዕ፡ ኤ<hi rend="rubric">ዎስ<supplied reason="omitted" resp="MV">ጣ</supplied>ቴዎስ<supplied reason="omitted" resp="MV">፡</supplied></hi>
+                           ከመ፡ እሌኒ፡ ንግሥት፡ መስቀለ፡ ኢየሱስ፡ ኀሣሤ፡ ወከመ፡ ቶማስ፡ ሐዋ<add place="interlinear">ር</add>ያ፡ አጽልዕተ፡ ገቦሁ፡ ገሣሤ፡ 
+                           ኤ<hi rend="rubric">ዎስጣቴዎስ፡</hi> ዜናዊ፡ ሃይማኖተ፡ ቅድ<surplus>ድ</surplus>ስት፡ ሥላሴ፡ አባ፡ አባ፡ አሣዕኒከ፡ ብናሴ፡ አባ፡ ከፋሌ፡ ባሕር፡ ዳግማይ፡ ሙሴ።
+                        </explicit>
+                        <explicit xml:lang="gez" type="supplication">
+                           <locus target="#77ra"/>
+                           ጸሎቱ፡ ወበረከቱ፡ ለአቡነ፡ ኤ<hi rend="rubric">ዎስጣቴዎስ<supplied reason="omitted" resp="MV">፡</supplied></hi> የሀሉ፡ ምስለ፡ ገብሩ፡ ገድለ፡
+                        </explicit>
+                     </msItem>
+                  </msContents>
+                  <physDesc>
+                     <objectDesc form="Other">
+                        <supportDesc>
+                           <support>
+                              <material key="parchment"/>
+                           </support>
+                           <extent>
+                              <measure unit="leaf">2</measure>
+                              <locus from="76" to="77"/>
+                              <measure unit="leaf" type="blank">1</measure>
+                              <locus target="#77v"/>
+                              <measure unit="quire">1</measure>
+                            <dimensions type="leaf" unit="mm" xml:id="leafdim2">
+                                 <height>207</height>
+                                 <width>175</width>
+                              </dimensions>
+                              <note corresp="#leafdim">Data on leaves dimensions are taken from <locus target="#76r"/>.</note>
+                           </extent>
+                           <foliation>The manuscript has been foliated in pencil in the lower margin of each folio.</foliation>
```

Comment:
Is it possible tp place this information for the whole book and not under its units?

Reply:
otherwiese, here again one needs to refer to the previous foliation

Reply:
I agree. The same remark could also be extended to
```
<support>
  <material key="parchment"/>
 </support>
```
which I repeated four times: once for the whole book and each time within each of the three units.
Do you think we can delete the last three occurrences?

Reply:
I would delete it.